bagit.py

```diff
@@ -1297,6 +1312,10 @@ def _make_parser():
                         help=_('Modify --validate behaviour to only test whether the bag directory'
                                ' has the number of files and total size specified in Payload-Oxum'
                                ' without performing checksum validation to detect corruption.'))
+    parser.add_argument('--completeness', action='store_true',
```

Comment:
That's what I was thinking – maybe even `--only-check-completeness` or `--validate-completeness-only`?